lib/version/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "version-sys"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
Don't change the version number. I do that myself when I publish a new version to crates.io